Microsoft/WebTemplateStudio
# angular nav not using routerLink

This change is to address1 of the generated angular code feedback from GitHub Issue: #665
Task:https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/WTS%20Team/Intern%20GitHub/Y19-VS/03

In Angular <a> tag should use "routerLink" rather than "href"

**How to test:**
1. generate an angular project with at least any two pages are selected 
2. start your application (npm install, npm start)
3. click links under nav bar should redirect to corresponding page. **(expected)**
